modules/Core/queries/debug_link.php

```diff
@@ -207,7 +207,7 @@
         'groups' => $groups,
         'config' => [
             'core' => array_filter(
-                $GLOBALS['config']['core'],
+                Config::read()['core'],
```

Comment:
Can't we do something like `Config::get('core');` here? Or is that not supported?

Would be easy enough to let `Config::get(...)` support returning `string|array`

Reply:
Good idea, then instead of adding a Config::all() method, just `Config::get('')` could be used

Reply:
Honestly just to scratch an itch, I'd make `Config::all()` just an alias to `Config::get('')`